source/reference/authentication.txt

```diff
@@ -0,0 +1,116 @@
+.. _c2c-authentication:
+
+=================================================
+Authentication Using Workload Identity Federation
+=================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+.. include:: /includes/mongosync-and-oidc.rst
+
+The source and destination clusters must use MongoDB 7.0 or later.
```

Comment:
I don’t _think_ this is true. It’s possible to migrate to/from self-managed clusters, which can be 6.0. (They wouldn’t use OIDC authn, of course.)

Reply:
Also: since the 7.0+ requirement is a server one rather than a mongosync requirement, does mongosync’s documentation even need to mention it? i.e., the fact that OIDC requires server 7.0 has nothing to do with mongosync; if a customer is migrating to/from an OIDC-authenticating cluster, that cluster is ipso facto 7.0+.